app/api/events.py

```diff
@@ -850,6 +850,7 @@ def query(self, view_kwargs):
                 Event.ends_at > current_time,
                 Event.state == 'published',
                 Event.privacy == 'public',
+                Event.tickets.any(and_(Ticket.deleted_at == None, Ticket.price > 0, Ticket.is_hidden == False)),
```

Comment:
Promoted events should be shown anyway

Reply:
Also price was an example. I don't think price is a requirement

Reply:
> Also price was an example. I don't think price is a requirement

ok